api/HTMLScriptElement.json

```
@@ -408,6 +408,49 @@
           }
         }
       },
+      "innerText": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/trusted-types/dist/spec/#dom-htmlscriptelement-innertext",
+          "tags": [
+            "web-features:trusted-types"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "83"
+            },
+            "chrome_android": "mirror",
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "135",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.security.trusted_types.enabled",
+                  "value_to_set": "true"
+                }
```

This should be same as `HTMLElement.innerText`:

```suggestion
              "version_added": "45"
```

After all, in Firefox 140 Stable, I can run this:

```js
document.createElement('script').innerText 
```

---

Hi @caugner 

FWIW it feels wrong, since the TT version on a script is effectively a different property. That said, this feels like a safer way to do it.